Configuration/Applications/python/ConfigBuilder.py

```diff
@@ -2232,6 +2232,8 @@ def prepare(self, doChecking = False):
             self.pythonCfgCode +="process.options.numberOfThreads=cms.untracked.uint32("+self._options.nThreads+")\n"
             self.pythonCfgCode +="process.options.numberOfStreams=cms.untracked.uint32("+self._options.nStreams+")\n"
             self.pythonCfgCode +="process.options.numberOfConcurrentLuminosityBlocks=cms.untracked.uint32("+self._options.nConcurrentLumis+")\n"
+            if self._options.nConcurrentLumis > 1:
+              self.pythonCfgCode +="if process.DQMStore: process.DQMStore.assertLegacySafe=cms.untracked.bool(False)\n"
```

Comment:
This should have probably been something like
```python
self.pythonCfgCode +="if hasattr(process, 'DQMStore'): process.DQMStore.assertLegacySafe=cms.untracked.bool(False)\n"
```
The latest IB (CMSSW_11_1/2020-03-16-1100) shows failures like
```
LHE input from article  18334
Note: this tool is DEPRECATED, use xrdfs instead.
customising the process with customiseWithTimeMemorySummary from Validation/Performance/TimeMemorySummary
Starting /data/cmsbld/jenkins/workspace/ib-run-relvals/cms-bot/monitor_workflow.py timeout --signal SIGTERM 9000  cmsRun -j JobReport1.xml  step1_NONE.py
----- Begin Fatal Exception 16-Mar-2020 12:51:11 CET-----------------------
An exception of category 'ConfigFileReadError' occurred while
   [0] Processing the python configuration file named step1_NONE.py
Exception Message:
 unknown python problem occurred.
AttributeError: 'Process' object has no attribute 'DQMStore'

At:
  step1_NONE.py(92): &lt;module&gt;

----- End Fatal Exception -------------------------------------------------
```
https://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc820/CMSSW_11_1_X_2020-03-16-1100/pyRelValMatrixLogs/run/1370.0_GluGluHToGG_M125_Pow_MINLO_NNLOPS_py8_13+GluGluHToGG_M125_Pow_MINLO_NNLOPS_py8_13+Hadronizer_TuneCUETP8M1_13TeV_powhegEmissionVeto_2p_HToGG_M125_13+DIGIUP15+RECOUP15+HARVESTUP15+NANOUP15Had/step1_GluGluHToGG_M125_Pow_MINLO_NNLOPS_py8_13+GluGluHToGG_M125_Pow_MINLO_NNLOPS_py8_13+Hadronizer_TuneCUETP8M1_13TeV_powhegEmissionVeto_2p_HToGG_M125_13+DIGIUP15+RECOUP15+HARVESTUP15+NANOUP15Had.log#/